uktrade/data-hub-frontend
# Add UI for the new interaction form

## Description of change

This is the first (out of 3) PR introducing a new interaction from.
The next PRs will include: Cypress tests and Redux integration.

The new route is not exposed to any users (yet).

## Test instructions

1. Go to https://data-hub-fro-new-intera-fely9k.herokuapp.com/interactions/create?company_id=5bb239e4-b227-4a84-bc2e-3420fc129447
2. Confirm if the form works the same as https://data-hub-fro-new-intera-fely9k.herokuapp.com/companies/5bb239e4-b227-4a84-bc2e-3420fc129447/interactions/create
